web_sdk/sdk_rewriter.dart

```diff
@@ -79,6 +79,7 @@ List<Replacer> generatePartsPatterns(String libraryName, bool isPublic) {
     // Remove library directives.
     AllReplacer(RegExp(r'\nlibrary .+;'), ''),
     // Remove imports/exports from all part files.
+    AllReplacer(RegExp(r"\nimport '.+'\n\s*if \(dart.library.html\) '.*';"), ''),
```

Comment:
can we not hardcode `dart.library.html` or maybe also allow for `dart.library.js_interop` as well, so our code doesn't mysteriously break when we migrate the code to not referencing dart.library.html? Bugs in the sdk rewriter are very hard to pinpoint :/

Reply:
Done.